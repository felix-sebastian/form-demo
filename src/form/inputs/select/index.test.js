import React from "react";
import Select from ".";
import { render } from "@testing-library/react";

it("displays the current value", () => {
  const { baseElement } = render(
    <Select value={0} options={["Foo", "Bar", "Baz"]} />
  );

  expect(baseElement).toMatchSnapshot();
});
