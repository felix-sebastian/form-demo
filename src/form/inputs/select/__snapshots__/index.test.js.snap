// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the current value 1`] = `
<body>
  <div>
    <div
      class="sc-fzXfLO eLTAmS"
    >
      <div
        class="sc-AykKF kLqURO"
        data-testid="input"
      >
        <input
          class="sc-AykKG cqNAOG sc-AykKH hepfOY"
          placeholder="Foo"
          value=""
        />
        <div
          class="sc-AykKI gsuSVD"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
            />
          </svg>
        </div>
      </div>
      <div
        class="sc-AykKJ fHucFO"
      />
    </div>
  </div>
</body>
`;
